internal/wrappers/grpcs/vorpal-grpc.go

```diff
@@ -0,0 +1,103 @@
+package grpcs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/checkmarx/ast-cli/internal/logger"
+	vorpalManagement "github.com/checkmarx/ast-cli/internal/wrappers/grpcs/protos/vorpal/managements"
+	vorpalScan "github.com/checkmarx/ast-cli/internal/wrappers/grpcs/protos/vorpal/scans"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+type VorpalWrapper struct {
+	grpcClient      *ClientWithTimeout
+	fullHostAddress string
+}
+
+const (
+	vorpalScanErrMsg = "Vorpal scan failed for file %s"
+	localHostAddress = "127.0.0.1:%d"
+	serviceName      = "VorpalEngine"
+)
+
+func NewVorpalWrapper(port int) *VorpalWrapper {
```

Comment:
interface missing for the new wrapper

Reply:
Why do we need an interface for the wrapper?